packages/react/src/date-picker/DatePicker.test.tsx

```diff
@@ -250,12 +246,36 @@ describe('DatePicker', () => {
 		expect(onChange).toHaveBeenLastCalledWith(date);
 
 		// The calendar button trigger should have an aria-label with the formatted display value
-		expect(await getCalendarTrigger()).toHaveAttribute(
+		expect(container.querySelector('button[type="button"]')).toHaveAttribute(
 			'aria-label',
 			`Change date, ${formattedDate}`
 		);
```

Comment:
You can lean into jest/testing-library to make this more robust:

```suggestion
		expect(
			screen.getByRole('button', { name: `Change date, ${formattedDate}` })
		).toBeVisible();
```

This gives us the added safety of knowing the element is visible ([see the checks here](https://github.com/testing-library/jest-dom?tab=readme-ov-file#tobevisible)) and that the `aria-label` is actually the accessible name (e.g. if someone has accidentally put an `aria-labelledby` property on it).

Reply:
OK done